<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>684</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>680</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>地铁换乘指南</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon/metro-shanghai.png</normaloff>:/icon/icon/metro-shanghai.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1030</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,11">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,2,0,5,2,32">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTransfer">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>地铁换乘指南</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout4" stretch="7,0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="labelStart">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>起点：</string>
          </property>
          <property name="buddy">
           <cstring>comboBoxStartLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStartLine">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="currentText">
           <string>选择路线</string>
          </property>
          <item>
           <property name="text">
            <string>选择路线</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxStartStation">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="currentText">
           <string>选择站点</string>
          </property>
          <item>
           <property name="text">
            <string>选择站点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout5" stretch="7,0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="labelDestination">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>目的地：</string>
          </property>
          <property name="buddy">
           <cstring>comboBoxDstLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDstLine">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>选择路线</string>
          </property>
          <item>
           <property name="text">
            <string>选择路线</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDstStation">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>选择站点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout6" stretch="0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>30</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radioButtonMinTransfer">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="text">
              <string>少换乘</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonMinTime">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
              </font>
             </property>
             <property name="text">
              <string>时间短</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonTransfer">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>搜索路线</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelTransferRoute">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>推荐路线：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowserRoute">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>580</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionuseHelp"/>
   </widget>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddLine"/>
   <addaction name="actionAddStation"/>
   <addaction name="actionAddConnect"/>
   <addaction name="actionAddByText"/>
   <addaction name="actionLineMap"/>
  </widget>
  <action name="actionAddLine">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/subway.png</normaloff>:/icon/icon/subway.png</iconset>
   </property>
   <property name="text">
    <string>线路(&amp;L)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionAddStation">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/station.png</normaloff>:/icon/icon/station.png</iconset>
   </property>
   <property name="text">
    <string>站点(&amp;S)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAddConnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/connect.png</normaloff>:/icon/icon/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接(&amp;C)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionAddByText">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/text.png</normaloff>:/icon/icon/text.png</iconset>
   </property>
   <property name="text">
    <string>文本方式(&amp;T)</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actiontoolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>toolBar</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="actionLineMap">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/map.png</normaloff>:/icon/icon/map.png</iconset>
   </property>
   <property name="text">
    <string>地铁图</string>
   </property>
   <property name="toolTip">
    <string>地铁网络线路图</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionWheel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/icon/mouse.png</normaloff>:/icon/icon/mouse.png</iconset>
   </property>
   <property name="text">
    <string>鼠标可缩放</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="actionuseHelp">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/icon/help.png</normaloff>:/icon/icon/help.png</iconset>
   </property>
   <property name="text">
    <string>使用说明</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
